src/test/java/pl/anasz/cinema/repositoriesTests/ScreeningRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package pl.anasz.cinema.repositoriesTests;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.anasz.cinema.enities.Movie;
+import pl.anasz.cinema.enities.Screening;
+import pl.anasz.cinema.repositories.ScreeningRepository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class ScreeningRepositoryTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+    @Autowired
+    private ScreeningRepository screeningRepository;
+
+    public ScreeningRepositoryTest() {
+    }
+
+    private Movie movie = new Movie();
+    private Screening screening = new Screening();
+
+    @Before
+    public void createTestEntities(){
+        movie.setTitle("Chilling adventures of Sabrina");
+        entityManager.persist(movie);
+
+        screening.setRoomId(3);
+        screening.setScreeningTime(LocalDateTime.of(2019,04,12,12,00,00));
+        screening.setMovie(movie);
+        entityManager.persist(screening);
+    }
+
+    @After
+    public void removeTestEntities(){
+        entityManager.remove(movie);
+        entityManager.remove(screening);
+    }
+
+    @Test
+    public void whenFindRoomIdForScreening_thenReturnRoom(){
```

Comment:
Taka nazwa metody miesza 2 konwencje nazewnicze. Możesz zaproponować lepszą nazwę?

Reply:
Jest to jedna z konwencji nazywania metod testowych, która dzięki temu w jasny sposób określa co jest testowane, tzn co ma być zrobione i jaki chcemy uzyskać wynik. Oczywiście można by zmienić tą nazwę jeśli chcielibyśmy ujednolicić konwencję na jednorodną, np. roomIdIsFoundForGivenScreeningId.